### ISSUE TITLE
Arabic bot is KO

### ISSUE THREAD

**Issue by aamarcha** [axa-group/nlp.js#134]
**Describe the bug**
I can't train a bot with arabic utterences.

**To Reproduce**
`const nlpManager = new NlpManager({ languages: ["ar"] });`
`nlpManager.addDocument("ar","سلام", "greeting");`
`nlpManager.train();`
 No error and train is not executed

But when I execute `nlpManager.process("ar", "سلام")` i get this error
`TypeError: this.current.length is not a function
    at ArabicStemmer.r_Checks1 (..  /nlp-js/node_modules/node-nlp/lib/nlp/stemmers/arabic-stemmer.js:302:52)`

**Expected behavior**
A trained bot

**Desktop:**
 - OS: Mac
- version "node-nlp": "^2.3.0",

**Comment by jesus-seijas-sp** [comment]
Solved in version 2.3.1 already available in npm.

Sample code:

```javascript
const { NlpManager } = require('./lib');

async function main() {
  const nlpManager = new NlpManager({ languages: ['ar'] });
  nlpManager.addDocument('ar', 'سلام', 'greeting');
  nlpManager.addDocument('ar', 'مرحبا', 'greeting')
  nlpManager.addDocument('ar', 'وداعا', 'bye');
  nlpManager.addDocument('ar', 'لا أجد مفتاحي', 'keys');
  await nlpManager.train();
  let result = await nlpManager.process('ar', 'سلام');
  console.log(result);
  result = await nlpManager.process('ar', 'مرحبا');
  console.log(result);
  result = await nlpManager.process('ar', 'وداعا');
  console.log(result);
  result = await nlpManager.process('ar', 'لا أجد مفتاحي');
  console.log(result);
}

main();
```

**Comment by jesus-seijas-sp** [comment]
Solved in 2.3.1
